client/python/armada_client/gen/event_typings.py

```diff
@@ -13,32 +13,47 @@ def get_all_job_event_classes():
     return [x for x in event_pb2.__dict__ if hasattr(getattr(event_pb2, x), "job_id")]
 
 
-def gen_file(states, classes):
+def get_job_states():
```

Comment:
Can you adding typing information here?  I'm not sure what name and value types are by looking over this code.

Reply:
Sure.